vllm/compilation/caching.py

```diff
@@ -134,11 +135,15 @@ def co_name(self):
         return "VllmSerializableFunction"
 
 
-def compilation_config_hash_factors(vllm_config: VllmConfig) -> list[str]:
+def compilation_config_hash_factors(
```

Comment:
This utility method is not the right abstraction, let's just inline it for now, or you can organize the code better if you have a better idea.